examples/upsert-resource/simple/.expected/config.yaml

```diff
@@ -0,0 +1 @@
+runCount: 2
```

Comment:
you don't need this, but default it runs twice.

Reply:
I will prioritize #275. Otherwise, we will be adding more and more examples using the old kpt.

Reply:
Retaining it in this PR.